features/step_definitions/manage_bookings_steps.rb

```diff
@@ -0,0 +1,18 @@
+Given('I have booking with') do |table|
```

Comment:
Generally the step definition should read `I have a booking with the follow properties` or `The following booking exists:` Decide a format and keep it consistent.